sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/geojson/GeoArray.java

```diff
@@ -0,0 +1,334 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.experimental.geojson;
```

Comment:
Why do we need `json` in this package name? Can we just use `geo` or `geometry`? Not sure if we should be including the serialization format for these types.

Reply:
As discussed offline `geojson` was chosen as geometries and spatial types extends outside of the scope of GeoJSON and that is what this implementation is targeting. cc: @pakrym 

For the serializers being in the package, that is done to scope feature work together in `azure-core-experimental`, once this functionality is merged into `azure-core` they'll exist in the serialization package.